code/modules/client/client procs.dm

```diff
@@ -383,6 +383,9 @@
 	if(prefs.toggles & DISABLE_KARMA) // activates if karma is disabled
 		if(establish_db_connection())
 			to_chat(src,"<span class='notice'>You have disabled karma gains.") // reminds those who have it disabled
+	else
+		if (establish_db_connection())
```

Comment:
Space needs to be removed from between `if` and `(....)

Reply:
Damn i'm so used to adding spacing for other projects

Reply:
Lol I have the opposite problem. In other projects I see the space but I'm so used to the nospace (as per this codebase's style guide) that it causes me the reverse

*shrug* thanks for fixing